ui/app/AppLayouts/Chat/CommunityComponents/CommunityUserList.qml

```diff
@@ -27,6 +27,31 @@ Item {
 
     property QtObject community: chatsModel.communities.activeCommunity
 
+    onCommunityChanged: {
+        proxyModel.clear()
+        for (let r = 0; r < community.members.rowCount(); r++) {
+            const pubKey = community.members.rowData(r, "address")
+            const nickname = appMain.getUserNickname(pubKey)
+            const identicon = community.members.rowData(r, "identicon")
+            const ensName = community.members.rowData(r, "ensName")
+            const name = !ensName.endsWith(".eth") && !!nickname ? nickname : Utils.removeStatusEns(ensName)
+            const statusType = chatsModel.communities.activeCommunity.memberStatus(pubKey)
+            const lastSeen = chatsModel.communities.activeCommunity.memberLastSeen(pubKey)
+            const lastSeenMinutesAgo = (currentTime / 1000 - parseInt(lastSeen)) / 60
+            const online = (pubKey === profileModel.profile.pubKey) || (lastSeenMinutesAgo < 7)
+
+            proxyModel.append({
+                pubKey: pubKey,
+                name: name,
+                identicon: identicon,
+                lastSeen: lastSeen,
+                statusType: statusType,
+                online: online,
+                sortKey: "%1%2".arg(online ? "A" : "B").arg(name)
```

Comment:
Why not doing this nim side, that would avoid us to go through the list and create a new one?
Wouldn't that be more performant?

Reply:
Will try re-implementing it that way today.

Reply:
@alaibe all done, please verify!